src/routes/Issues/components/Issues.tsx

```diff
@@ -32,7 +32,7 @@ export interface IParams {
   experience_needed?: string[] | string;
   language?: string[] | string;
   type?: string[] | string;
-  ordering?: string;
+  ordering?: string[] | string;
```

Comment:
This should remain a string, we can't have multiple ordering values

Reply:
okay